src/Sentry.Unity/SentryUnity.cs

```diff
@@ -24,19 +28,54 @@ public static void Init(Action<SentryUnityOptions> sentryUnityOptionsConfigure)
         /// <summary>
         /// Initializes Sentry Unity SDK while providing an options object.
         /// </summary>
-        /// <param name="sentryUnityOptions">The options object.</param>
+        /// <param name="options">The options object.</param>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public static void Init(SentryUnityOptions sentryUnityOptions)
+        public static void Init(SentryUnityOptions options)
         {
-            SentryOptionsUtility.TryAttachLogger(sentryUnityOptions);
-            if (sentryUnityOptions.ShouldInitializeSdk())
+            SentryOptionsUtility.TryAttachLogger(options);
+            if (options.ShouldInitializeSdk())
             {
-                sentryUnityOptions.DiagnosticLogger?.LogDebug(sentryUnityOptions.ToString());
-                var sentryDotNet = SentrySdk.Init(sentryUnityOptions);
+                // On Standalone, we disable cache dir in case multiple app instances run over the same path.
+                // Note: we cannot use a named Mutex, because Unit doesn't support it. Instead, we create a file with `FileShare.None`.
+                // https://forum.unity.com/threads/unsupported-internal-call-for-il2cpp-mutex-createmutex_internal-named-mutexes-are-not-supported.387334/
+                if (ApplicationAdapter.Instance.Platform == RuntimePlatform.WindowsPlayer && options.CacheDirectoryPath != null)
+                {
+                    try
+                    {
+                        _lockFile = new FileStream(Path.Combine(options.CacheDirectoryPath, "sentry-unity.lock"), FileMode.OpenOrCreate,
+                                FileAccess.ReadWrite, FileShare.None);
+                    }
+                    catch (Exception ex)
+                    {
+                        options.DiagnosticLogger?.Log(SentryLevel.Warning, "An exception was thrown while trying to " +
```

Comment:
Can we do that instead?

```suggestion
                        options.DiagnosticLogger?.LogWarning("An exception was thrown while trying to " +
```

Reply:
couldn't in the public code. It should work now.